app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html

```diff
@@ -26,166 +26,120 @@
+                </div>
+                <div class="column-two-thirds">
+                    <p>
+                    @if(btiApp.otherInformation.isDefined) {
+                        @btiApp.otherInformation.get
+                    }else {
+                        @messages("answer.none")
                     }
-                }
-            }
-
-            @if(btiApp.reissuedBTIReference.isDefined) {
-                <hr/>
-                <h4 id="re-issue-heading" class="heading-small">This application is for a re-issue for BTI reference #@btiApp.reissuedBTIReference.get</h4>
-            }
-
-            @if(btiApp.envisagedCommodityCode.isDefined) {
-                <hr/>
-                <div id="app-details-envisaged-code" class="grid-row">
-                    <div class="column-one-third">
-                        <p>@messages("case.application.suggested-commodity-code")</p>
-                    </div>
-                    <div class="column-two-thirds">
-                        <p>@btiApp.envisagedCommodityCode.get</p>
-                    </div>
+                    </p>
                 </div>
-            }
+            </div>
 
-            <div class="grid-row">
+            <div id="app-details-sending-samples" class="grid-row">
                 <div class="column-one-third">
-                    <p>@messages("case.application.goods-description")</p>
+                    <p>@messages("case.application.sending-samples")</p>
                 </div>
                 <div class="column-two-thirds">
-                    <p>@btiApp.goodDescription</p>
+                    <p>@btiApp.sampleToBeProvided</p>
                 </div>
             </div>
 
-            @if(btiApp.confidentialInformation.isDefined) {
-                <div id="app-details-confidential-info" class="grid-row">
-                    <div class="column-one-third">
-                        <p>@messages("case.application.confidential-information")</p>
-                    </div>
-                    <div class="column-two-thirds">
-                        <p>@btiApp.confidentialInformation.get</p>
-                    </div>
+            <div id="app-details-reissue-application" class="grid-row">
+                <div class="column-one-third">
+                    <p>@messages("case.application.reissue-application")</p>
                 </div>
-            }
-
-            @if(btiApp.relatedBTIReference.isDefined) {
-                <div id="app-details-related-reference" class="grid-row">
-                    <div class="column-one-third">
-                        <p>@messages("case.application.related-bti-reference")</p>
-                    </div>
-                    <div class="column-two-thirds">
-                        <p>@btiApp.relatedBTIReference.get</p>
-                    </div>
+                <div class="column-two-thirds">
+                    <p>
+                        @if(btiApp.reissuedBTIReference.isDefined) {
```

Comment:
You can do this in a `getOrElse`
`btiApp.reissuedBTIReference.getOrElse(messages("answer.no"))`